examples/sycl/pvc/pvc_gemm.cpp

```diff
@@ -333,9 +333,9 @@ int main(int argc, const char** argv)
           ElementOutput,
           cutlass::gemm::TagToStrideC_t<LayoutD>,
           FusionCallBacks,
-          XE_2D_U32x8x16x1x1_LD_N,
+          XE_2D_U32x8x16_LD_N,
           void, void,
-          XE_2D_U32x8x16x1x1_ST_N,
+          XE_2D_U32x8x16_ST_N,
           void, void>;
```

Comment:
Could you update the GEMM configuration for the benchmarks too (benchmarks/pvc/gemm_configuration.hpp)? 

For running the benchmarks you can do: 

./benchmarks/benchmarks --config_file=/path-to-cutlass/benchmarks/pvc/input.in

Reply:
I want to know the purpose of benchmark, performance or functionality?

Reply:
performance tuning will be our next task

Reply:
That's ok, but the changes in this PR will break the benchmark code. Just make sure it compiles. You can tune for performance in a follow PR as planned.

Reply:
OK, I will fix it ASAP